Cython/Compiler/Symtab.py

```diff
@@ -2705,12 +2706,13 @@ def declare_inherited_cpp_attributes(self, base_class):
                                            modifiers=base_entry.func_modifiers,
                                            utility_code=base_entry.utility_code)
             entry.is_inherited = 1
-        for base_entry in base_scope.type_entries:
+        for base_entry in base_scope.type_entries + base_scope.inherited_type_entries:
             if base_entry.name not in base_templates:
                 entry = self.declare_type(base_entry.name, base_entry.type,
                                           base_entry.pos, base_entry.cname,
-                                          base_entry.visibility)
+                                          base_entry.visibility, defining=0)
```

Comment:
Probably `defining=False` instead of `0` (a lot of Cython code predates the bool I believe, so is written with 0, but there's no need to keep it like that with new code)

Reply:
Understood, that'll end up in the next update of this PR.